### PR TITLE
[WIP] Integrate wfm module with imported UI modules

### PR DESCRIPTION
**Pull request by paolobueno**
ping @wtrocki for progress

### CODE REVIEW

**Comment by paolobueno** [comment]
companion to https://github.com/feedhenry-raincatcher/raincatcher-angularjs/pull/5

File: client/wfm/src/executor/Executor.ts [hunk 1]
```diff
@@ -1,5 +1,14 @@
+import * as Promise from 'bluebird';
 import {ProcessInstance} from '../process-instance/ProcessInstance';
 import {Process} from '../process/Process';
+import {Task} from '../task/Task';
+
+export interface TaskMoveResponse {
```
Comment (wtrocki) [inline client/wfm/src/executor/Executor.ts]: Can we work with documentation first aproach and add ts-docs compatiblie documentation when creating classes.  I'm not sure what is the purpose of this class for the moment.

**Comment by wtrocki** [comment]
@paolobueno is that still actuall? Maybe we can integrate this into another PR ?

**Comment by wtrocki** [comment]
Closing as outdated. Feel free to reopen if needed.

**Pull request closed without merging**